### PR TITLE
Start Google Rise application for 2016

### PR DESCRIPTION
**Pull request by jkwok91**
**Under discussion with @zachlatta**

Closes https://github.com/hackclub/meta/issues/445.

### CODE REVIEW

File: applications/google_rise_2016.md [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+
+**Detailed project description (include all activities):**
+
+We would like RISE to fund the continued expansion of Hack Club. Specifically,
+we're looking to bring someone on full-time to lead the development of our
+curriculum with the community.
+
+To expand on what Hack Club does:
+
+[TODO]
```
Comment (jkwok91) [inline applications/google_rise_2016.md]: - plan and write curriculum
- maintain community guidelines
- onboard new clubs
- periodic check-ins with existing clubs over calls
- hack camp / other events
- monitor slack [almost] 24/7

File: applications/google_rise_2016.md [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+**Does this project have a post-engagement strategy to keep students connected
+with computer science? If yes, please describe.**
+
+We are the post-engagement strategy. Our clubs meet weekly or bi-weekly
+throughout the school year and we've received partnership interest from a
+variety of computer science summer programs wanting their students to start Hack
+Clubs after their program ends.
+
+For the summer, when our clubs are out of session, we hope to partner with the
+aforementioned summer programs and eventually run our own hackathons to
```
Comment (jkwok91) [inline applications/google_rise_2016.md]: What about connection w collegiate hackathons/community?

File: applications/google_rise_2016.md [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+
+**Mission Statement**
+
+Our mission is to provide every student, regardless of ethnicity, gender, or
+socioeconomic status, the opportunity to experience the joys of hacking because
+we believe coding is one of the most empowering things someone can learn. If
+we're able to accomplish this, not only does that mean a larger and more diverse
+technology workforce, but it means living in a more empowered world where more
+people create the change they want to see. And that is a world we want to live
+in
```
Comment (MaxWofford) [inline applications/google_rise_2016.md]: Period

File: applications/google_rise_2016.md [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+1,500+
+
+**Demographics of youth reached by organization in 2015**
+
+_Please use approximate percentages to list the demographics of the youth that
+your organization served last year, such as age ranges, race/ethnicity, gender
+etc. For example, 25% boys, 75% girls._
+
+- 32% women (compare with 20% female AP CS test takers)
+- 37% minorities in tech (compare with 13% African American/Hispanic AP CS test
```
Comment (MaxWofford) [inline applications/google_rise_2016.md]: These two statistics (minorities in tech and African American/Hispanic) are not the same demographic.

File: applications/google_rise_2016.md [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+**What were the organization's major achievements in the past year?**
+
+We've taken Hack Club from an idea attempted at a single high school to the
+beginning of an international movement in 70 schools across 16 states and 7
+countries (just started a club in Pakistan!). We've helped start clubs at
+schools in a variety of socioeconomic situations, from schools that are
+upper-middle class with access to lots of resources to schools that have been
+labeled "dropout factories".
+
+We've our content significantly and are now at the point where we have a "club
```
Comment (MaxWofford) [inline applications/google_rise_2016.md]: Missing a word here?

File: applications/google_rise_2016.md [hunk 6]
```diff
@@ -0,0 +1,414 @@
...
+**Who will be leading this project? Include the title and relevant experience.**
+
+Zachary Latta, Executive Director of Hack Club.
+
+A former club leader himself, Latta began Hack Club after he realized how much
+of an impact finding peers and forming a coding community made on him in terms
+of confidence, mentorship, and impetus to improve. Having run Hack Club for over
+a year now, Latta has experience writing curriculum and running workshops.
+
+what am i writing
```
Comment (paked) [inline applications/google_rise_2016.md]: :")

File: applications/google_rise_2016.md (hunk 4 above)
Comment (jkwok91) [inline applications/google_rise_2016.md]: Yea, I wrote this but I just pulled it off the website... anyone got better stats?

Comment (MaxWofford) [inline applications/google_rise_2016.md]: /cc @zachlatta

Comment (zachlatta) [inline applications/google_rise_2016.md]: We can just say 7x industry average. I'll have to find the source for that.

Comment (zachlatta) [inline applications/google_rise_2016.md]: I changed it to say `37% black/Hispanic`

File: applications/google_rise_2016.md [hunk 7]
```diff
@@ -0,0 +1,459 @@
...
+
+**Brief description of the organization**
+
+Hack Club helps high school students start coding clubs. We provide prospective
+club leaders with open-source materials for starting clubs and with a supportive
+community of leaders around the globe.
+
+Our clubs have gone on to host hackathons with hundreds of participants, build
+projects to impact their local communities, and bring coding education to
+traditionally underserved and underrepresented communities.
```
Comment (jkwok91) [inline applications/google_rise_2016.md]: Communities 2x?

File: applications/google_rise_2016.md [hunk 8]
```diff
@@ -0,0 +1,459 @@
...
+
+**What makes your organization unique, innovative, and/or sets you apart from
+other groups doing similar work?**
+
+Our clubs and content are built and run by students, for students.
+
+By having high schoolers lead clubs, we're able to bypass the red tape typically
+associated with bringing computer science to a school or a community. Where it
+can take years to start a new computer science program, starting a Hack Club can
+take as little as two weeks. Our leaders don't need to find their own space (the
```
Comment (jkwok91) [inline applications/google_rise_2016.md]: "Where" -> "Whereas" or "while" http://www.beatthegmat.com/while-vs-whereas-vs-where-t8737.html

File: applications/google_rise_2016.md (hunk 7 above)
Comment (jkwok91) [inline applications/google_rise_2016.md]: "underserved and underrepresented _populations_"? Hmm..

File: applications/google_rise_2016.md [hunk 9]
```diff
@@ -0,0 +1,459 @@
...
+To expand on what Hack Club does:
+
+We provide everything high schoolers need to start amazing coding clubs: meeting
+guidelines, curriculum, template slideshows, and a supportive community of
+leaders around the globe.
+
+Club meetings are run in pairs and are like mini-hackathons: the first meeting
+in the pair kicks off with a workshop and the second culminates in demos.
+Members are shipping something in every meeting, whether from a workshop or from
+what they built for the mini-hackathon. By the end of the first meeting,
```
Comment (jkwok91) [inline applications/google_rise_2016.md]: Wait, what's the difference (for my own understanding)? "From a workshop" vs "from a mini-hackathon"? Isn't the mini-hackathon an extension of the workshop? Also, is pointing out the distinction important?

File: applications/google_rise_2016.md [hunk 10]
```diff
@@ -0,0 +1,459 @@
...
+everyone has built and deployed their first website from a workshop. By the end
+of the second, they've expanded on it and demoed it. By the end of the third,
+everyone's built a website that can call and text their phone (again, from a
```
Comment (jkwok91) [inline applications/google_rise_2016.md]: What is the purpose of mentioning the next workshop? It sounds a bit redundant / too similar to the first workshop. Either chop this or make it stand out more, like "everyone learns to *work with an API* to build something that involves *the phone*" (rephrase).

File: applications/google_rise_2016.md [hunk 11]
```diff
@@ -0,0 +1,459 @@
...
+- Members are winning hackathons and other competitions
+- Members are hosting their own events (hackathons, meetups, etc)
+
+**How do you plan to measure and evaluate your results?**
+
+We'll continue to measure our primary KPIs: the number of clubs and the stories
+we're hearing from club leaders and members.
+
+Additionally, we'll begin measuring attendance of Hack Clubs week-to-week and
+their attrition rates.
```
Comment (jkwok91) [inline applications/google_rise_2016.md]: Would it be helpful to say something about GitHub activity / online presence?

File: applications/google_rise_2016.md [hunk 12]
```diff
@@ -0,0 +1,459 @@
...
+Fellowships would be sponsored by companies and foundations. We've received
+preliminary interest from companies interested in sponsoring this sort of
+program.
+
+**Does this project have a post-engagement strategy to keep students connected
+with computer science? If yes, please describe.**
+
+For many programs, we are the post-engagement strategy. Our clubs meet weekly or
+bi-weekly throughout the school year and we've received partnership interest
+from a variety of computer science summer programs wanting their students to
```
Comment (jkwok91) [inline applications/google_rise_2016.md]: Is it helpful to include which programs? Or is that jumpin the gun cuz the partnerships are not concrete yet?

File: applications/google_rise_2016.md (hunk 7 above)
Comment (zachlatta) [inline applications/google_rise_2016.md]: Maybe change the first "communities" to "neighborhoods"?

File: applications/google_rise_2016.md (hunk 9 above)
Comment (zachlatta) [inline applications/google_rise_2016.md]: - From a workshop workshop = what you build directly from the workshop (ex. the exact website you get from the workshop)
- From a mini-hackathon = what you build from expanding on what you started in the workshop (ex. 
http://jsbin.com/vanaxa/2)

Distinction probably isn't important. Want to reword to remove it?

File: applications/google_rise_2016.md (hunk 10 above)
Comment (zachlatta) [inline applications/google_rise_2016.md]: Primary purpose is to make sure they understand the format of meetings. If you don't think mentioning the second workshop is necessary, let's kill it.

File: applications/google_rise_2016.md (hunk 11 above)
Comment (zachlatta) [inline applications/google_rise_2016.md]: Maybe, we don't really have concrete steps to start doing this though. I say keep it simple for now.

File: applications/google_rise_2016.md (hunk 12 above)
Comment (zachlatta) [inline applications/google_rise_2016.md]: Partnerships aren't concrete yet, don't want to jump the gun.

File: applications/google_rise_2016.md (hunk 9 above)
Comment (jkwok91) [inline applications/google_rise_2016.md]: "Members are shipping something in every meeting, whether it's the workshop project idea or an expansion of that idea." ?

File: applications/google_rise_2016.md (hunk 11 above)
Comment (jkwok91) [inline applications/google_rise_2016.md]: :+1:

File: applications/google_rise_2016.md (hunk 12 above)
Comment (jkwok91) [inline applications/google_rise_2016.md]: :+1:

**Pull request merged**